src/EntryPoints/Rest/SaveMappingsApi.php

```diff
@@ -44,17 +47,16 @@ public function getParamSettings(): array {
 
 	/**
 	 * @inheritDoc
+	 * @return array<string, array<string, mixed>>
 	 */
-	public function getBodyValidator( $contentType ): BodyValidator {
-		if ( $contentType !== 'application/json' ) {
-			throw new HttpException(
-				"Unsupported Content-Type",
-				415,
-				[ 'content_type' => $contentType ]
-			);
-		}
-
-		return new JsonBodyValidator( [] );
+	public function getBodyParamSettings(): array {
```

Comment:
To get `getValidatedBody()` working as before, I have no choice but to change the schema for the POST method, that's why I bumped the version of the REST API.

Reply:
I don't see a corresponding change in the JS. Did you verify it still works?

Reply:
I have updated the JS side as well. 
However, I haven't run a smoke test on this yet, will do it later if no one gets to it first.